generics/services/certificate.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * name : certificate.js
+ * author : Vishnu
+ * Date : 07-Oct-2022
+ * Description : Sunbird-RC certificate api.
+ */
+
+//dependencies
+const request = require('request');
+const CERTIFICATE_SERVICE_URL = process.env.CERTIFICATE_SERVICE_URL;
+const ML_PROJECT_URL = `https://${process.env.SERVICE_NAME}:${process.env.APPLICATION_PORT}`;
```

Comment:
@VISHNUDAS-tunerlabs this would be http, also can be set via env and default value given via the envVariables.js